server/sonar-ce/src/main/java/org/sonar/ce/container/CePluginRepository.java

```diff
@@ -63,10 +65,11 @@ public CePluginRepository(ServerFileSystem fs, PluginLoader loader) {
 
   @Override
   public void start() {
-    Loggers.get(getClass()).info("Load plugins");
+    LOGGER.info("Load plugins");
     for (File file : listJarFiles(fs.getInstalledPluginsDir())) {
       PluginInfo info = PluginInfo.create(file);
       pluginInfosByKeys.put(info.getKey(), info);
+      LOGGER.info("Loaded plugin {} [{}]", info.getName(), info.getKey());
```

Comment:
also, at this point in the code, the plugin is not loaded, only PluginInfo is read from the jar metadata (correct me if I'm wrong). The loading actually occurs on line 74. The log could be misleading.

Reply:
the additional commit it's debatable. I'm in favour of squashing commits when merging into master. Splitting commits is required for easier code review. In the current PR, I don't think it would make a difference. I hope it was trivial for you to review the few touched lines.
Anyway I'm going to improve the log and take the opportunity to extract a commit.